src/adagrams.js

```diff
@@ -1,15 +1,133 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const numTiles = 10;
+  const lettersInHand = [];
+  const currentLetterPool = [...listOfLetters]
```

Comment:
Nice used of the spread operator. 

While JavaScript's automatic semicolon insertion (ASI) will add a semicolon after line 4, our curriculum prefers to explicitly put them after every statement for clarity and to avoid potential issues. This can be team dependent so if you're working on JS, check in with your teammates to see how they do it.

```suggestion
  const currentLetterPool = [...listOfLetters];
```